web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -176,25 +180,38 @@
 
   {#if !api.isSharedLink && people.length > 0}
     <section class="px-4 py-4 text-sm">
-      <h2>PEOPLE</h2>
+      <div class="flex w-full items-center justify-between">
+        <h2>PEOPLE</h2>
+        <div class={people.reduce((acc, p) => acc || p.isHidden, false) ? '' : 'hidden'}>
+          <CircleIconButton
+            title="Show hidden people"
+            icon={showingHiddenPeople ? mdiEyeOff : mdiEye}
+            padding="0"
+            on:click={() => (showingHiddenPeople = !showingHiddenPeople)}
+          />
+        </div>
+      </div>
 
       <div class="mt-4 flex flex-wrap gap-2">
```

Comment:
What do you think about this ?
```suggestion
      <div class="flex h-10 w-full items-center justify-between">
        <h2>PEOPLE</h2>
        {#if people.some((person) => person.isHidden)}
          <CircleIconButton
            title="Show hidden people"
            icon={showingHiddenPeople ? mdiEyeOff : mdiEye}
            padding="1"
            on:click={() => (showingHiddenPeople = !showingHiddenPeople)}
          />
        {/if}
      </div>

      <div class="mt-2 flex flex-wrap gap-2">
```

Reply:
Yep, that looks cleaner. I don't do much web dev so I don't really know what I'm doing here.

I've given it a test locally and it works the same so merged this into the PR.